src/components/main/ItemScreens/SettingsItemScreen.tsx

```diff
@@ -0,0 +1,52 @@
+import { Navigate, useNavigate, useOutletContext } from 'react-router';
+
+import { ArrowBack } from '@mui/icons-material';
+import { Box, Button } from '@mui/material';
+
+import { useShortenURLParams } from '@graasp/ui';
+
+import { PERMISSIONS_EDITION_ALLOWED } from '@/config/constants';
+import { useBuilderTranslation } from '@/config/i18n';
+import { ITEM_ID_PARAMS, buildItemPath } from '@/config/paths';
+import { BUILDER } from '@/langs/constants';
+
+import ItemSettings from '../../item/settings/ItemSettings';
+import Navigation from '../../layout/Navigation';
+import WrappedItemScreen from './WrappedItemScreen';
+import { OutletType } from './type';
+
+const ItemSettingsScreen = (): JSX.Element => {
+  const itemId = useShortenURLParams(ITEM_ID_PARAMS);
+  const { item, permission } = useOutletContext<OutletType>();
+
+  const navigate = useNavigate();
+  const { t: translateBuilder } = useBuilderTranslation();
+
+  const enableEditing = permission
+    ? PERMISSIONS_EDITION_ALLOWED.includes(permission)
+    : false;
+
+  if (enableEditing) {
+    return (
+      <Box py={1} px={2}>
+        <Navigation />
+        <Box px={2}>
+          <Button
+            startIcon={<ArrowBack fontSize="small" />}
+            variant="text"
+            onClick={() => navigate(buildItemPath(itemId))}
+          >
+            {translateBuilder(BUILDER.BACK)}
+          </Button>
+          <ItemSettings item={item} />
+        </Box>
+      </Box>
+    );
+  }
+  return <Navigate to={buildItemPath(itemId)} replace />;
```

Comment:
Maybe add a comment that this redirects the user to the item if he doesn't have the permission to access this page. Same for all other pages.